### PR TITLE
RHDEVDOCS-3571 RN for Pipelines 1.6.1

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.9`, `enterprise-4.10`
- **JIRA issues**: [RHDEVDOCS-3571 RN for Pipelines 1.6.1](https://issues.redhat.com/browse/RHDEVDOCS-3571)
- **Preview pages**: [Release notes for Red Hat OpenShift Pipelines General Availability 1.6.1](https://deploy-preview-39887--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes#release-notes-1-6-1_op-release-notes)
- **SME review**: @concaf    
- **Peer-review**: @Preeticp

### CODE REVIEW

**Review by concaf — APPROVED** [review]

File: modules/op-release-notes-1-6.adoc [hunk 1]
```diff
@@ -244,4 +244,23 @@ time="2021-11-04T13:05:26Z" level=error msg="exit status 127"
...
+
+The following improvements are made in {pipelines-title} 1.6.1:
+
+* The `SSL_CERT_DIR` environment variable (`/tekton-custom-certs`) set by {pipelines-title} will not override the following default system directories with certificate files:
+** `/etc/pki/tls/certs`
+** `/etc/ssl/certs`
+** `/system/etc/security/cacerts`
+// https://issues.redhat.com/browse/SRVKP-1687
+
+* The Horizontal Pod Autoscaler can manage the replica count of deployments controlled by the {pipelines-title} Operator. From this release onward, the {pipelines-title} Operator will not reset the replica count of deployments managed by it, if the count is changed by an end user or an on cluster agent. However, the replicas will be reset when you upgrade the {pipelines-title} Operator.
```
Comment (Preeticp) [inline modules/op-release-notes-1-6.adoc:262]: Suggest: 'on-cluster agent' to increase clarity.
Suggest: From this release onward, if the count is changed by an end user or an on cluster agent, the {pipelines-title} Operator will not reset the replica count of deployments managed by it. 
That way user will read the second part only if it is relevant for them.

Comment (Preeticp) [inline modules/op-release-notes-1-6.adoc:262]: Suggest: 'on-cluster agent' to increase clarity.
Suggest: From this release onward, if the count is changed by an end user or an on cluster agent, the {pipelines-title} Operator will not reset the replica count of deployments managed by it. 
That way an user cna simply skim the second part if it is not relevant for them.

**Review by Preeticp — COMMENTED** [review]
@sounix000 a couple of minor suggestions otherwise LGTM.

**Pull request merged**